cdap-api/src/main/java/co/cask/cdap/api/schedule/Schedule.java

```diff
@@ -53,4 +54,40 @@ public String getDescription() {
   public String getCronEntry() {
     return cronEntry;
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+
+    Schedule schedule = (Schedule) o;
+
+    if (!cronEntry.equals(schedule.cronEntry) || !description.equals(schedule.description)
```

Comment:
Fixed. Please take a look again and let me know if any formatting changes are needed.